caps/models.py

```diff
@@ -189,15 +189,15 @@ def powers(self):
 
         if (
             self.country != self.NORTHERN_IRELAND
-            and self.authority_type != "COMB"
+            and self.authority_type not in ["COMB", "SRA"]
```

Comment:
I realise there's some benefit in splitting this up by conceptually similar authority types but equally we could just add "NMD" to the list and only have two conditions here, and below and I think it would be fine.

Reply:
Yep.